src/components/project/query.helpers.ts

```diff
@@ -44,6 +44,14 @@ export function projectListFilter(query: Query, filter: ProjectFilters) {
       node('node'),
     ]);
   }
+
+  if (filter.pinned) {
```

Comment:
I think it makes sense to have `pinned == false` not match any pinned projects. That's why I had `pinned != null`, so both true and false could be handled.
What do you think?

Reply:
This [commit](https://github.com/SeedCompany/cord-api-v3/pull/1734/commits/f41c61cc1edc0b17919ed844442087ddff9118b5) will handle this function.